rs/consensus/src/idkg/test_utils.rs

```diff
@@ -1648,6 +1654,14 @@ pub(crate) fn schnorr_algorithm(algorithm: AlgorithmId) -> SchnorrAlgorithm {
     }
 }
 
+pub(crate) fn fake_vetkd_key_id() -> VetKdKeyId {
+    VetKdKeyId::from_str("Bls12_381:some_key").unwrap()
+}
+
+pub(crate) fn fake_vetkd_master_public_key_id() -> MasterPublicKeyId {
+    MasterPublicKeyId::VetKd(fake_vetkd_key_id())
+}
+
```

Comment:
With the added panic above, I don't think these functions are used
```suggestion
```

Reply:
Removed.